crates/wysiwyg/src/dom/unicode_string.rs

```diff
@@ -142,4 +197,124 @@ impl<S: UnicodeStr + ?Sized> UnicodeStrExt for S {
     fn len(&self) -> usize {
         self.as_ref().len()
     }
+
+    /// Assuming [index] is a boundary between graphemes, returns a pair with the previous and next
+    /// graphemes, if present.
+    fn find_graphemes_at(
```

Comment:
Could `find_graphemes_at` take the `pos` in its natural units (i.e. UTF-16 code units for us), and then call `u8_mapped_indexes` inside itself? This would make the interface much less confusing I think.

Reply:
I'll take a look.